dts/bindings/stepper/gpio-stepper-controller.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO Stepper Controller cluster for darlington transistor arrays or dual H-bridge
+
+compatible: "gpio-steppers"
+
+child-binding:
+  description: GPIO Controller for stepper motor
+  include:
+    - name: stepper-controller.yaml
+      property-allowlist:
+        - micro-step-res
+
+  properties:
+    gpios:
+      type: phandle-array
+      required: true
+      description: |
+        The gpio pin array on which the stepper inputs are to be connected
+        stepper {
+              compatible = "gpio-steppers";
+              status = "okay";
+              motor:motor {
+                gpios = <&gpioa 9 GPIO_ACTIVE_HIGH>,
```

Comment:
you are better off putting the example in the description of the binding as a whole, the on e at line 4

Reply:
done